src/routes/safe/components/Apps/components/AddAppForm/index.tsx

```diff
@@ -93,14 +98,24 @@ const AddApp = ({ appList, closeModal }: AddAppProps): ReactElement => {
               <Icon size="sm" type="externalLink" color="primary" />
             </Link>
           </AppDocsInfo>
+
           <AppUrl appList={appList} />
 
           {/* Fetch app from url and return a SafeApp */}
-          <AppInfoUpdater onAppInfo={setAppInfo} />
+          <AppInfoUpdater onAppInfo={setAppInfo} setIsLoading={setIsLoading} />
 
           <AppInfo>
-            <Img alt="Token image" height={55} src={appInfo.iconUrl} />
-            <StyledTextFileAppName label="App name" readOnly value={appInfo.name} onChange={() => {}} />
+            {isLoading ? (
+              <StyledLoader size="md" />
```

Comment:
When the loader appears, the UI jumps a bit.
Probably need a container with the same size as the app image.

Reply:
https://www.loom.com/share/292540cee3d5486ca38c854ca7be0e32
mmm... I can not see the jump, can you?

Reply:
should be fixed now